backend/db/migrate/20231123093623_change_tourism_and_recreation_to_sustainable_tourism.rb

```diff
@@ -1,12 +1,17 @@
 class ChangeTourismAndRecreationToSustainableTourism < ActiveRecord::Migration[7.0]
   def up
     Project.where(category: "tourism-and-recreation").update_all(category: "sustainable-tourism")
+
     [Investor, ProjectDeveloper].each do |model|
-      model.find_each do |record|
+      model.where("categories @> ?", "{tourism-and-recreation}").find_each do |record|
```

Comment:
Almost there :). It should be something like this:
```
new_categories_for_all_records = model.where("categories @> ?", "{tourism-and-recreation}").pluck(:id, :categories).map do |id, categories|
  new_categories = categories.map do |category|
    category == "tourism-and-recreation" ? "sustainable-tourism" : category
  end
  {id: id, categories: new_categories}
end
model.upsert_all new_categories_for_all_records, unique_by: :id
```

Problem is that you are triggering `upsert_all` inside cycle so it gets called once per every record. It needs to go outside so you collect all data inside cycle and then outside of it call just once `upsert_all` per model.